locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -368,15 +364,17 @@ useful for the user to be alerted to an error before the event loop is
 allowed to continue. Here is the previous example using `process.nextTick()`:
 
 ```js
-function someAsyncApiCall (callback) {
+let bar;
```

Comment:
Why are you initializing the variable up here instead of assigning it directly as before?